components/automate-cli/pkg/verifyserver/services/statusservice/statusservice.go

```diff
@@ -1,19 +1,141 @@
 package statusservice
 
 import (
+	"regexp"
+	"strings"
+
+	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/constants"
 	"github.com/chef/automate/components/automate-cli/pkg/verifyserver/models"
+	"github.com/chef/automate/lib/logger"
+	"github.com/gofiber/fiber"
 )
 
 type IStatusService interface {
-	GetServices() []models.ServiceDetails
+	GetServices() (*[]models.ServiceDetails, error)
+}
+
+type StatusService struct {
+	ExecuteShellCommandFunc func(cmd string) ([]byte, error)
+	Log                     logger.Logger
+}
+
+func NewStatusService(executeShellCommand func(cmd string) ([]byte, error), log logger.Logger) IStatusService {
```

Comment:
NewStatusService should return object of StatusService not interface

Reply:
done